conans/test/util/tools_test.py

```diff
@@ -157,24 +157,31 @@ def system_package_tool_test(self):
             spt.update()
             self.assertEquals(runner.command_called, "sudo yum check-update")
 
+            os_info.linux_distro = "opensuse"
+            spt = SystemPackageTool(runner=runner, os_info=os_info)
```

Comment:
This test is failing in OSX, because the creation of the tool check first ``os_info.is_macos`` which happens to be true. Maybe the zypper tool could be checked first, or make sure in the test to put ``os_info.is_macos`` to False